store/src/java/com/zimbra/cs/service/account/SearchGal.java

```diff
@@ -71,14 +71,27 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
     
     private static Element searchGal(ZimbraSoapContext zsc, Account account, Element request) throws ServiceException {
         
-        // if searhc by ref is requested, honor it
+        // if search by ref is requested, honor it
         String ref = request.getAttribute(AccountConstants.A_REF, null);
         
-        // otherwise require a query
+        String galSearchQuery = null;
+        
         String name = null;
-        if (ref == null) {
-            name = request.getAttribute(AccountConstants.E_NAME);
-        }
+        
+		if (ref == null) {
```

Comment:
indentation looks off for this if block, please correct it.

Reply:
done